testing/trino-testing/src/main/java/io/trino/testing/FaultTolerantExecutionConnectorTestHelper.java

```diff
@@ -28,6 +28,8 @@ private FaultTolerantExecutionConnectorTestHelper() {}
                 .put("query.initial-hash-partitions", "5")
                 .put("fault-tolerant-execution-target-task-input-size", "10MB")
                 .put("fault-tolerant-execution-target-task-split-count", "4")
+                // to trigger spilling
+                .put("exchange.deduplication-buffer-size", "1kB")
```

Comment:
can we have some test (just single use-case) to verify that spilling actually happens if this value is low, and does not if it is high?

Reply:
I tried to cover it with unit tests. I may try to extract statistics for the deduplication buffer and check whether spilledBytes is not 0

Reply:
It's actually not that easy to extract operator statistics. There's a unit test that verifies whether spilling actually takes places. Verifying it here is more of verify whether the configuration parameter has been correctly passed to the buffer, as it is extremely unlikely that the hundreds of queries executed are not actually exceeding the 1kB limit.   I wonder if it's worth it?

Reply:
Not pushing too much. But getting stats should not be that hard with `AbstractTestQueryFramework#assertQueryStats`

Reply:
The stats about whether spilling occur will be stored in the `OperatorInfo` within the `OperatorStats`. To get the right `OperatorStats` you somehow need to know the stageId, pipelineId and the operatorId for the `ExchangeOperator` that uses `DeduplicationBuffer`. While it is possible to figure (or even hardcode) this is not the only challenge.

For `SELECT` queries the `DeduplicationBuffer` is used in the `ExchangeClient` created to fetch final query results (in `Query.java`) which is not attached to any operator. The stats from this buffer are not available at all. 

So it leaves only the `INSERT` queries to play with. But for `INSERT` queries the data size produced by table writers is not well predictable. It may vary based on seemingly unrelated changes to the connector. 

In theory it is possible to set the limit to something extremely low, e.g.: `10B`. But then it will cause all queries to spill, and what I wanted to achieve here is to exercise both spilling and non spilling scenarios.